src/main/java/practice/Sachin/Program11_FactorialCalculation.java

```diff
@@ -0,0 +1,15 @@
+package practice.Sachin;
+
+public class Program11_FactorialCalculation {
+
+    // Factorial Calculation using for loop
+    public static void main(String[] args){
+
+        int number = 5;//It is the number to calculate factorial
+        int fact = 1;
+        for(int i=1; i<=number; i++){
+            fact = fact * i;
+        }
+        System.out.println("Factorial of "+number+" is : "+fact);
+    }
```

Comment:
public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        System.out.println("check factorial");
        int a = sc.nextInt();
        int b=1;
        for (int i=a;i>1;i--)
        {
           a=a*(i-1);
        }
       System.out.println(a);
    }

Reply:
You can use this method too